autogpts/autogpt/prompt_settings.yaml

```diff
@@ -4,8 +4,6 @@ constraints: [
   'You are unable to interact with physical objects. If this is absolutely necessary to fulfill a task or objective or to complete a step, you must ask the user to do it for you. If the user refuses this, and there is no other way to achieve your goals, you must terminate to avoid wasting time and energy.'
 ]
 resources: [
-  'Internet access for searches and information gathering.',
-  'The ability to read and write files.',
```

Comment:
Removed the resources that are now provided bu the `web_search` and `file_manager` components.